cmd/record.go

```diff
@@ -303,6 +308,8 @@ func (r *Record) GetCmd() *cobra.Command {
 
 	recordCmd.Flags().StringP("command", "c", "", "Command to start the user application")
 
+	recordCmd.Flags().DurationP("recordTimer", "t", 0, "Timer to stop keploy recorder after a specified time")
```

Comment:
why is the shorthand of the flag -t? That can confuse the user, either start the name with t or change the shorthand to the first alphabet of the full flag name.

Reply:
how about  -rt?

Reply:
Heyy! so i removed the shorthand entirely and kept it as --recordTimer , keeping the shorthand as -r would lead to ambiguity, So please check